sklearn/metrics/pairwise.py

```diff
@@ -1000,9 +1000,9 @@ def paired_cosine_distances(X, Y):
 
 def paired_distances(X, Y, *, metric="euclidean", **kwds):
```

Comment:
You are missing a docstring for the `**kwds` argument

Reply:
Thanks @glemaitre.  Added the docstring for that now.

Reply:
So we decided to not deprecate and remove this parameter.
We can document it and mentioned that it is not used at the moment.